frontera/contrib/scrapy/converters.py

```diff
@@ -26,7 +28,7 @@ def to_frontier(self, scrapy_request):
         if callable(eb):
             eb = _find_method(self.spider, eb)
 
-        scrapy_meta = scrapy_request.meta
+        scrapy_meta = dict_to_bytes(scrapy_request.meta, keys_only=True)
```

Comment:
I agree with @Preetwinder, correct me if I'm wrong @sibiryakov but over within frontera, you want to encode everything in bytes so that it Python version agnostic; important for a distributed system which may have Python 2 and Python 3 on different nodes. I'm going to go ahead and reimplement the above to deal with preserving all type information. The only thing I haven't decided upon is what happens when the meta is passed from Scrapy running Py2 to Scrapy running Py3 and vice versa. 

e.g. for `scrapy_meta`, Python 3 string (`unicode`) -> `to_frontier` -> (`bytes`) -> `from_frontier` -> Python 2 (`str` or `unicode`?)